### ISSUE TITLE
Loss decreases but the acc is always 0?

### ISSUE THREAD

**Issue by Piaoyyt** [ronghuaiyang/arcface-pytorch#93]
In the trainig progress, the trainig loss is decreasing but the accuracy on the training dataset is always zero. I don't know why this happens?

**Comment by deanmark** [comment]
The default margin **m** is very high. Try starting with **m**=0, see if this improves the accuracy. If this helps, try slowly increasing **m** until you obtain a satisfactory result.

**Comment by Jia-Xueyu** [comment]
> 

Hello, I have the same question. Do you solve it now?

**Comment by Jia-Xueyu** [comment]
> In the trainig progress, the trainig loss is decreasing but the accuracy on the training dataset is always zero. I don't know why this happens?

Hello, I have the same question. Do you solve it now?

**Comment by Rui-coder** [comment]
Are you sure that the Loss Model is in the training state? Auto-grad-discrese should also apply to the loss model !!!

**Comment by yulongPan** [comment]
请问解决了吗
